ArmVirtPkg/ArmVirtPkg.ci.yaml

```diff
@@ -56,7 +56,8 @@
             "ArmPlatformPkg/ArmPlatformPkg.dec",
             "PcAtChipsetPkg/PcAtChipsetPkg.dec",
             "SecurityPkg/SecurityPkg.dec",
-            "ShellPkg/ShellPkg.dec"  #Is this ok?
+            "ShellPkg/ShellPkg.dec",  #Is this ok?

```

Comment:
can you scope this to UEFI APPLICATION or does you firmware code really need this?

Reply:
This is needed in the `PlatformHasAcpiDtDxe.inf` DXE driver I pulled in for ARM platform. This driver is using a library header file originally defined in OvmfPkg..